<a href="https://colab.research.google.com/github/AdrianCCT/Fundamental-of-Statistics-of-DA/blob/main/Probability_Q1_%26_Q2.ipynb" target="_parent"><img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"/></a>

# Question 1

In [3]:
p= 0.04
#1 is average number of properties wihtout acess to fresh water
# n*p=1

In [5]:
n=1/p
n

25.0

Answer is 25

# Question 2

In [10]:
from scipy.stats import norm, binom

In [11]:
#p = probability of P(X>11)
mean= 10.5
std_dev= 2
X = 11

In [12]:
p= norm.sf(X,mean,std_dev)
p

0.4012936743170763

In [13]:
# now we can calcualte the Probability that Exactly 2 out of 4 Years Have Withdrawals Greater than 11 bcm:

In [28]:
# totla number of years is 4
n=4
# we want to find probability for 2 year out of 4
k=2
# we have already calcuated proabability for p=P(X>11)=0.4012936743170763


In [22]:
P2Yof4Y=binom.pmf(k,n,p)
P2Yof4Y

0.34634073294108836

In [27]:
r=100*round(P2Yof4Y, 4)
percentage=f"{r}%"
percentage

'34.63%'

Probability that in  2 out of 4 years withdrawal was greater than 11 bcm is approx. 34.63%.